shared/src/common/interfaces/register/register-payload.interface.ts

```diff
@@ -0,0 +1,14 @@
+import { RegisterPayloadKey, UserType, UserSex } from '../../enums'
+
+export interface IRegisterPayload {
+  [RegisterPayloadKey.NAME]: string;
+  [RegisterPayloadKey.SURNAME]: string;
+  [RegisterPayloadKey.SEX]: UserSex;
+  [RegisterPayloadKey.BIRTH_DATE]: Date;
+  [RegisterPayloadKey.EMAIL]: string;
+  [RegisterPayloadKey.PASSWORD]: string;
+  [RegisterPayloadKey.RETYPE_PASSWORD]: string;
+  [RegisterPayloadKey.PHONE]: string;
+  [RegisterPayloadKey.TYPE]: UserType;
+  [RegisterPayloadKey.IMAGE_PATH]: string;
+}
```

Comment:
```suggestion
interface IRegisterPayload {
  [RegisterPayloadKey.NAME]: string;
  [RegisterPayloadKey.SURNAME]: string;
  [RegisterPayloadKey.SEX]: UserSex;
  [RegisterPayloadKey.BIRTH_DATE]: Date;
  [RegisterPayloadKey.EMAIL]: string;
  [RegisterPayloadKey.PASSWORD]: string;
  [RegisterPayloadKey.RETYPE_PASSWORD]: string;
  [RegisterPayloadKey.PHONE]: string;
  [RegisterPayloadKey.TYPE]: UserType;
  [RegisterPayloadKey.IMAGE_PATH]: string;
}

export type { IRegisterPayload };
```
Let's do export only at the bottom of the file